src/business_logic/execution/objects.rs

```diff
@@ -252,7 +252,7 @@ pub struct TransactionExecutionContext {
     pub(crate) n_emitted_events: u64,
```

Comment:
Why do you underscore this fields? @pefontana

Reply:
Ohh sorry, some merge conflicts
Solved